### PR TITLE
Please review P0-avatar-badges

### PR DESCRIPTION
**Pull request by ianuragsharma**

### CODE REVIEW

**Comment by Bibin-Antony** [comment]
kindly add a preview image of the component.

File: badge.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.badge {
+  position: relative;
+  width: max-content;
+}
+.badge-icon > i {
+  font-size: 3rem;
+}
+.badge > div {
+  bottom: 5px;
```
Comment (Bibin-Antony) [inline badge.css]: px to  em

**Comment by ishitakeshawani** [comment]
@ianuragsharma Please create the proper folder structure for your components and add files to it.

**Review by ishitakeshawani — APPROVED** [review]
You can add a square shape too in the avatar if you want.

**Review by ishitakeshawani — CHANGES_REQUESTED** [review]
you missed adding ./utility.css
so badges not looking proper.
please add its screenshot too.

File: badge.css (hunk 1 above)
Comment (ianuragsharma) [reply badge.css]: Done

**Comment by ianuragsharma** [comment]
Screenshot 2022-02-03 180755
image

**Review by ishitakeshawani — APPROVED** [review]
Looks good

**Review by rohanmathur91 — APPROVED** [review]
Looks good.

**Pull request merged**